specification/2.0/schema/material.schema.json

```diff
@@ -5,28 +5,79 @@
     "description" : "The material appearance of a primitive.",
     "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
     "properties": {
-        "technique" : {
-            "allOf" : [ { "$ref" : "glTFid.schema.json" } ],
-            "description" : "The ID of the technique.",
-            "gltf_detailedDescription" : "The ID of the technique.  If this is not supplied, and no extension is present that defines material properties, then the primitive should be rendered using a default material with 50% gray emissive color."
-        },
-        "values" : {
-            "type" : "object",
-            "description" : "A dictionary object of parameter values.",
-            "properties" : {
-            },
-            "additionalProperties" : {
-                "$ref" : "material.values.schema.json"
-            },
-            "default" : {},
-            "gltf_detailedDescription" : "A dictionary object of parameter values.  Parameters with the same name as the technique's parameter override the technique's parameter value."
+        "baseColorFactor" : {
+            "type" :"array",
+            "description" : "The material's base color factor.",
+            "default" : [1,1,1,1],
+            "gltf_detailedDescription" : "The RGBA components of the base color (RGB) of the material. The fourth component (A) is the opacity of the material."
+        },
+         "baseColorTexture" : {
+            "extends" : "number",
+            "description" : "The index of the base color texture."
+        },
+        "baseColorTexCoord" : {
+            "extends" : "number",
+            "description" : "The set index of the base color texture's TEXCOORD attribute to be used for texture coordinate mapping."
+        },
+        "metallicFactor" : {
+            "type" : "number",
+            "description" : "The metallic-ness the material (1 for metals, 0 for non-metals).",
+            "default" : 1
+        },
```

Comment:
Is there a reason that metallicFactor doesn't have a minimum and maximum value like roughnessFactor?